examples/dt/uni05epsilon/control-plane/nncp/values.yaml

```diff
@@ -0,0 +1,212 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: network-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  openstack-operator-image: "quay.io/openstack-k8s-operators/openstack-operator-index:latest"
+
+  node_0:
+    name: master-0
+    internalapi_ip: 172.17.0.5
+    tenant_ip: 172.19.0.5
+    ctlplane_ip: 192.168.122.10
+    storage_ip: 172.18.0.5
+  node_1:
+    name: master-1
+    internalapi_ip: 172.17.0.6
+    tenant_ip: 172.19.0.6
+    ctlplane_ip: 192.168.122.11
+    storage_ip: 172.18.0.6
+  node_2:
+    name: master-2
+    internalapi_ip: 172.17.0.7
+    tenant_ip: 172.19.0.7
+    ctlplane_ip: 192.168.122.12
+    storage_ip: 172.18.0.7
+
+  ctlplane:
+    dnsDomain: ctlplane.example.com
+    subnets:
+      - allocationRanges:
+          - end: 192.168.122.120
+            start: 192.168.122.100
+          - end: 192.168.122.200
+            start: 192.168.122.150
+        cidr: 192.168.122.0/24
+        gateway: 192.168.122.1
+        name: subnet1
+    prefix-length: 24
+    iface: enp6s0
+    mtu: 9000
```

Comment:
Why MTU 9000 on the control plane?

See https://github.com/openstack-k8s-operators/architecture/pull/159#discussion_r1571107733

Reply:
Hello, I have no idea, this is a draft, not ready for review at the moment; we all copy-pasted what we were told to use as a baseline – please ask the question there: https://github.com/openstack-k8s-operators/architecture/pull/83/files

Reply:
No more 9000 :fire:

Reply:
Ugh... I had it changed, then I messed it again after splitting the stages.
All other Uni DTs have the mtu set to 9000.
So in the end I am not sure which one should be there :dog: 
```
./examples/dt/uni06zeta/control-plane/nncp/values.yaml:44:    mtu: 9000
./examples/dt/uni06zeta/control-plane/nncp/values.yaml:108:    mtu: 9000
./examples/dt/uni01alpha/control-plane/nncp/values.yaml:44:    mtu: 9000
./examples/dt/uni01alpha/control-plane/nncp/values.yaml:108:    mtu: 9000
./examples/dt/uni02beta/control-plane/nncp/values.yaml:44:    mtu: 9000
./examples/dt/uni02beta/control-plane/nncp/values.yaml:108:    mtu: 9000
./examples/dt/uni02beta/control-plane/nncp/values.yaml:137:    mtu: 9000
./examples/dt/uni04delta/control-plane/nncp/values.yaml:45:    mtu: 9000
./examples/dt/uni04delta/control-plane/nncp/values.yaml:109:    mtu: 9000
./examples/dt/uni04delta/control-plane/nncp/values.yaml:139:    mtu: 9000
./examples/dt/uni04delta/edpm-pre-ceph/nodeset/values.yaml:81:        storage_mtu: 9000
./examples/dt/uni04delta/edpm-pre-ceph/nodeset/values.yaml:82:        storage_mgmt_mtu: 9000
./examples/dt/uni04delta/values.yaml:92:        storage_mtu: 9000
./examples/dt/uni04delta/values.yaml:93:        storage_mgmt_mtu: 9000
```

Reply:
Some UniDTs have a reason for setting MTU 9000 on the storage networks (perf benefits). I see no reason to set it on the control-plane network unless it's there are storage networks which use the control-plane network as their trunk. So I recommend you just remove MTU 9000 from the epsilon DT in all networks OR restrict it only to storage.

Reply:
Just to be precise – not some, ALL current UniDTs have it to 9000 for both ctl and storage:
```
[sdatko@polluks architecture]$ grep -nri 9000 examples/dt/ examples/va/hci --before-context=13 | grep -e storage -e ctl -e mtu | grep -v dns
examples/dt/uni06zeta/control-plane/nncp/values.yaml-31-  ctlplane:
examples/dt/uni06zeta/control-plane/nncp/values.yaml:44:    mtu: 9000
examples/dt/uni06zeta/control-plane/nncp/values.yaml-99-  storage:
examples/dt/uni06zeta/control-plane/nncp/values.yaml:108:    mtu: 9000
examples/dt/uni01alpha/control-plane/nncp/values.yaml-31-  ctlplane:
examples/dt/uni01alpha/control-plane/nncp/values.yaml:44:    mtu: 9000
examples/dt/uni01alpha/control-plane/nncp/values.yaml-99-  storage:
examples/dt/uni01alpha/control-plane/nncp/values.yaml:108:    mtu: 9000
examples/dt/uni02beta/control-plane/nncp/values.yaml-31-  ctlplane:
examples/dt/uni02beta/control-plane/nncp/values.yaml:44:    mtu: 9000
examples/dt/uni02beta/control-plane/nncp/values.yaml-99-  storage:
examples/dt/uni02beta/control-plane/nncp/values.yaml:108:    mtu: 9000
examples/dt/uni02beta/control-plane/nncp/values.yaml-128-  storagemgmt:
examples/dt/uni02beta/control-plane/nncp/values.yaml:137:    mtu: 9000
examples/dt/uni04delta/control-plane/nncp/values.yaml-32-  ctlplane:
examples/dt/uni04delta/control-plane/nncp/values.yaml:45:    mtu: 9000
examples/dt/uni04delta/control-plane/nncp/values.yaml-100-  storage:
examples/dt/uni04delta/control-plane/nncp/values.yaml:109:    mtu: 9000
examples/dt/uni04delta/control-plane/nncp/values.yaml-130-  storagemgmt:
examples/dt/uni04delta/control-plane/nncp/values.yaml:139:    mtu: 9000
examples/dt/uni04delta/edpm-pre-ceph/nodeset/values.yaml:81:        storage_mtu: 9000
examples/dt/uni04delta/edpm-pre-ceph/nodeset/values.yaml:82:        storage_mgmt_mtu: 9000
examples/dt/uni04delta/values.yaml:92:        storage_mtu: 9000
examples/dt/uni04delta/values.yaml:93:        storage_mgmt_mtu: 9000
examples/va/hci/control-plane/nncp/values.yaml-31-  ctlplane:
examples/va/hci/control-plane/nncp/values.yaml:44:    mtu: 9000
examples/va/hci/control-plane/nncp/values.yaml-97-  storage:
examples/va/hci/control-plane/nncp/values.yaml:106:    mtu: 9000
examples/va/hci/control-plane/nncp/values.yaml-126-  storagemgmt:  # used on RHEL, not used on OpenShift
examples/va/hci/control-plane/nncp/values.yaml:135:    mtu: 9000
examples/va/hci/edpm-pre-ceph/nodeset/values.yaml:95:        storage_mtu: 9000
examples/va/hci/edpm-pre-ceph/nodeset/values.yaml:96:        storage_mgmt_mtu: 9000
```

Reply:
We can deal with 9000 then in another PR.

Reply:
Ok, just for the record – I set it to 1500 and it went fine, no more 9000 now in this PR.